lib/SampleService/core/config.py

```diff
@@ -207,15 +221,43 @@ def get_validators(url: str) -> MetadataValidatorSet:
     :returns: A set of metadata validators.
     '''
     # TODO VALIDATOR make validator CLI
+
     try:
-        with _urllib.request.urlopen(url) as res:
-            cfg = _yaml.safe_load(res)
+        config_asset = None
+        if url:
+            config_url = url
+        elif not repo_path:
+            raise ValueError(f'No metadata validator config URL or repo path.')
+        else:
+            repo = _Github(login_or_token=token).get_repo(repo_path)
+            releases = [rel for rel in repo.get_releases() if not rel.prerelease]
+            if not releases:
+                raise ValueError(f'No releases found in validator config repo {repo_path}')
+            latest_release = releases[0] # max(releases, key=lambda rel: rel.created_at)
+            assets = latest_release.get_assets()
+            if not assets:
+                raise ValueError(f'No assets found in validator config repo {repo_path}')
+            config_asset = next((a for a in assets if a.name=='metadata_validation.yml'), None)
```

Comment:
should "metadata_validation.yml" perhaps be assignable?

Reply:
That is probably a good idea. I wasn't sure how many new config variables we wanted, but this makes sense for sure.